INFO:root:Navigated to polybuzz login page
INFO:root:User has confirmed login
INFO:root:Navigating back to the main polybuzz page
INFO:root:Begin selecting the character: Character_1
INFO:root:Refreshed the page.
ERROR:root:Error occurred while selecting the character: Message: 

ERROR:root:Error occurred during conversation generation: Message: 

INFO:root:Page refreshed successfully
INFO:root:Begin selecting the character: Character_1
INFO:root:Refreshed the page.
ERROR:root:Error occurred while selecting the character: Message: 

ERROR:root:Error occurred during conversation generation: Message: 

INFO:root:Page refreshed successfully
INFO:root:Begin selecting the character: Character_1
INFO:root:Refreshed the page.
ERROR:root:Error occurred while selecting the character: Message: 

ERROR:root:Error occurred during conversation generation: Message: 

INFO:root:Page refreshed successfully
INFO:root:Begin selecting the character: Character_1
INFO:root:Refreshed the page.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10709ae90>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/274c2a09-8e0c-444e-bdc3-bdb28010d6d6
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107110590>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/274c2a09-8e0c-444e-bdc3-bdb28010d6d6
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107110bd0>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/274c2a09-8e0c-444e-bdc3-bdb28010d6d6
